docs/windows.md

```diff
@@ -80,6 +81,8 @@ It is necessary to always:
 - Carry the OVMF firmware in the `--kernel` option
 - Add `kvm_hyperv=on` to the `--cpus` option
 
+In cases where the host processor supports address space &gt; 39 bits, it might be necessary to limit the address space. It can be done by appending the option `max_phys_bits=X` to the `--cpus` switch, where `X` is the number of bits to be supported. Windows is guaranteed to support at least a 39-bit address space.
```

Comment:
I would avoid saying "guaranteed to support" because we haven't checked the code. The limitation may also change in the future.

I would be fair to say "we've tested up to 39-bit address space".

Reply:
Ack, rephrased the suggested way.

Thanks.